src/types/appointment.ts

```diff
@@ -0,0 +1,12 @@
+export type Appointment = {
+  user: any // @TODO: Figure out correct type

```

Comment:
`user` can be `firebase.User.UserRecord` type. Right?

Reply:
I found the correct type sir, it is `EntityReference` from `firecms`